### ISSUE TITLE
The certificate::get_course_time method contains an error

### ISSUE THREAD

**Issue by scalimax** [mdjnelson/moodle-mod_customcert#403]
It seems to me that the total time calculation contains a mistake. The $CFG->sessiontimeout is stored in the mdl_config table as seconds (i.e. if I set 60 minutes as session timeout, the stored value is 3600). 
In the method get_course_time the fetched value is multiplied by 60 before comparing it to the current $delay, that's calculated as seconds (see https://github.com/mdjnelson/moodle-mod_customcert/blob/a9f83a2579ba7b3d12aaf327c02581347ab364ef/classes/certificate.php#L220).
This overrates the final $totaltime that's returned and of course it makes the test on the requested spent time fail.
In other words a user can get the certificate even if the requested spent time is not reached.

**Comment by mdjnelson** [comment]
Thanks for the issue. I swear I came across the problem before and had to add the multiplier. Anyways, this function needs a rewrite. Adding it as one of the things to do before release.

**Comment by mdjnelson** [comment]
If you visit 'Dashboard' > 'Site administration' > 'Server' > 'Session handling' and look at the setting 'sessiontimeout' you can select the number of hours. I chose 9 hours then looked at the value `$CFG->sessiontimeout` which set it to the number of seconds (32400). Multiplying this by 60 doesn't make sense. I need to go through the logic more but I think you may be right.

**Comment by mdjnelson** [comment]
Ah, the multiplication I was confused with was on `view.php` not in this function. I am removing the multiplication.

**Comment by mdjnelson** [comment]
Fixed. Thanks.
